src/components/molecules/AppPageHeader.tsx

```diff
@@ -3,12 +3,14 @@ interface PageHeaderProps {
     description?: string | null;
 }
 
-export default ({title, description}: PageHeaderProps) => {
+const PageHeaderProps = ({title, description}: PageHeaderProps) => {
```

Comment:
Why is this called `PageHeaderProps` - it's a component, it should be called `AppPageheader`

Reply:
Completely missed. Nice catch!
Fixed.